website/static/js/project-organizer.js

```diff
@@ -376,6 +377,13 @@ var tbOptions = {
     hScroll : 300,
     filterTemplate : function() {
         var tb = this;
+        function resetFilter () {
+            $osf.trackClick('myProjects', 'filter', 'clear-search');
+            tb.filterText('');
+            tb.resetFilter.call(tb);
+            tb.updateFolder(allTopLevelProjectsCache(), tb.treeData);
+            $('.db-poFilter>input').val('');
```

Comment:
This might be related to a redraw problem, technically tb.filterText('') should clear it. Let's leave this for later to not block the rest of the code.